pom.xml

```diff
@@ -43,6 +44,7 @@
     <embedded.redis.version>0.6</embedded.redis.version>
     <wiremock.version>2.26.3</wiremock.version>
     <elasticsearch.version>6.4.0</elasticsearch.version>
+    <newrelic-api.version>5.0.0</newrelic-api.version>
```

Comment:
Need to sync this with newrelic agent version in java docker base images. I think it was not 5.0.0 now.

Reply:
update to 5.11.0